docs/use_a_template.md

```diff
@@ -17,7 +17,7 @@ This imports the specified `manifest.yml` into an instance of InfluxDB running o
 
 If you don't want to download the manifest file locally, you can point to its remote location using the `--url` flag, for example:
 ```
- influx pkg --url https://raw.githubusercontent.com/influxdata/community-templates/master/template/manifest.yml
+ influx apply --url https://raw.githubusercontent.com/influxdata/community-templates/master/template/manifest.yml
  ```
```

Comment:
This section should be updated to say that the `-f` flag supports both file paths and URLS.